x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/common_attributes.gen.ts

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Common Defend Insights Attributes
+ *   version: not applicable
+ */
+
+import { z } from '@kbn/zod';
+
+import { NonEmptyString, User } from '../common_attributes.gen';
+import { Replacements, ApiConfig } from '../conversations/common_attributes.gen';
+
+/**
+ * A Defend insight event
+ */
+export type DefendInsightEvent = z.infer<typeof DefendInsightEvent>;
```

Comment:
I’m wondering if `@elastic/security-defend-workflows` should be made `CODEOWNERS` of the following paths:

```
x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights
x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights
x-pack/plugins/elastic_assistant/server/routes/defend_insights
```

At the time of this writing, the current `CODEOWNERS` are:

## Files by Code Owner

### elastic/security-defend-workflows

* x-pack/plugins/security_solution/common/endpoint/constants.ts

### elastic/security-generative-ai

* x-pack/packages/kbn-elastic-assistant-common/constants.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/common_attributes.gen.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/common_attributes.schema.yaml
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/get_defend_insight_route.gen.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/get_defend_insight_route.schema.yaml
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/get_defend_insights_route.gen.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/get_defend_insights_route.schema.yaml
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/index.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/post_defend_insights_route.gen.ts
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/defend_insights/post_defend_insights_route.schema.yaml
* x-pack/packages/kbn-elastic-assistant-common/impl/schemas/index.ts
* x-pack/plugins/elastic_assistant/common/anonymization/index.ts
* x-pack/plugins/elastic_assistant/server/__mocks__/defend_insights_schema.mock.ts
* x-pack/plugins/elastic_assistant/server/__mocks__/request.ts
* x-pack/plugins/elastic_assistant/server/__mocks__/request_context.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/field_maps_configuration.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/get_defend_insight.test.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/get_defend_insight.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/helpers.test.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/helpers.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/index.test.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/index.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/defend_insights/types.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_service/index.test.ts
* x-pack/plugins/elastic_assistant/server/ai_assistant_service/index.ts
* x-pack/plugins/elastic_assistant/server/lib/langchain/helpers.ts
* x-pack/plugins/elastic_assistant/server/lib/telemetry/event_based_telemetry.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/get_defend_insight.test.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/get_defend_insight.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/get_defend_insights.test.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/get_defend_insights.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/helpers.test.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/helpers.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/index.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/post_defend_insights.test.ts
* x-pack/plugins/elastic_assistant/server/routes/defend_insights/post_defend_insights.ts
* x-pack/plugins/elastic_assistant/server/routes/index.ts
* x-pack/plugins/elastic_assistant/server/routes/register_routes.ts
* x-pack/plugins/elastic_assistant/server/routes/request_context_factory.ts
* x-pack/plugins/elastic_assistant/server/types.ts

### elastic/security-solution

* x-pack/plugins/security_solution/common/endpoint/constants.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/errors.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/get_events/get_file_events_query.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/get_events/index.test.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/get_events/index.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/index.test.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/index.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/output_parsers/incompatible_antivirus.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/output_parsers/index.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/prompts/incompatible_antivirus.ts
* x-pack/plugins/security_solution/server/assistant/tools/defend_insights/prompts/index.ts
* x-pack/plugins/security_solution/server/assistant/tools/index.ts